registry/src/api.rs

```diff
@@ -15,12 +15,25 @@
 //! # Buffrs Registry API
 
 use crate::context::Context;
+use crate::proto::buffrs::registry::registry_server::{RegistryServer};
+use tonic::transport::Server;
+use eyre::Result;
 
 mod grpc;
 
 impl Context {
     /// Launch API
-    pub async fn launch_api(&self) {
-        todo!()
+    pub async fn launch_api(&self) -> Result<()>  {
+        let ctx = self.clone();
+
+        let server = Server::builder()
+        .add_service(RegistryServer::new(ctx));
```

Comment:
```suggestion
            .add_service(RegistryServer::new(ctx));
```

(`cargo fmt` should be able to fix this :))